Tests/UI/BillingForm/View/BillingFormTextFieldViewTests.swift

```diff
@@ -24,11 +24,6 @@ class BillingFormTextFieldViewTests: XCTestCase {
 
     func testTextFieldContainerStyle() {
         XCTAssertEqual(view.textFieldContainer.backgroundColor, style.textfield.backgroundColor)
-        XCTAssertEqual(view.textFieldContainer.layer.borderColor, style.textfield.normalBorderColor.cgColor)
-        XCTAssertEqual(view.textFieldContainer.layer.cornerRadius, style.textfield.cornerRadius)
-        XCTAssertEqual(view.textFieldContainer.layer.borderWidth, style.textfield.borderWidth)
-        XCTAssertEqual(view.textFieldContainer.layer.cornerRadius, style.textfield.cornerRadius)
-        XCTAssertEqual(view.textFieldContainer.layer.borderWidth, style.textfield.borderWidth)
```

Comment:
Would these tests not work anymore, just having to update the `style` path to the new wrapper object for border?